src/content/1.7/code/agda/snippet13.agda

```diff
@@ -0,0 +1,4 @@
+record Functor (F : Set → Set) : Set where
+  field fmap : (A → B) → F A → F B
```

Comment:
Change `F`, `A`, and `B` to `f`, `a`, and `b` (resp.) so these symbols match with the ones mentioned in the surrounding prose.... but see comment below (re. snippet32).